### PR TITLE
feat(temporary banners)

### PR DESCRIPTION
**Pull request by esinx**
Screenshot 2023-10-14 at 5 55 30 PM
Screenshot 2023-10-14 at 5 55 10 PM
Screenshot 2023-10-14 at 5 54 00 PM

### CODE REVIEW

File: frontend/review/src/components/Announcement.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useRef, useState } from "react"
+
```
Comment (andyjiang3) [inline frontend/review/src/components/Announcement.js:2]: Does `pcx-shared-components` not work on pcr?

**Review by andyjiang3 — APPROVED** [review]
lgtm for temp banners! we currently have separate banners for each apps (ex. [pca](https://github.com/pennlabs/penn-courses/blob/1d0fc3ec697914a7aa5c63f53249cf40469d67ad/frontend/alert/pages/index.tsx#L214)) that would need to be deleted after migration to banner for all web products (can do this in a diff pr).

**Comment by andyjiang3** [comment]
Also looks like you need to `yarn clean` to fix the linting issue

File: frontend/review/src/components/Announcement.js (hunk 1 above)
Comment (esinx) [reply frontend/review/src/components/Announcement.js:2]: I assumed that from the [README](https://github.com/pennlabs/penn-courses/tree/master/frontend). Let me trying moving that into shared components instead, but I'm not sure how using TypeScript in PCR will play out here!

**Pull request merged**